Source/VisualRecognitionV4/Models/BaseTrainingDataObjects.swift

```diff
@@ -0,0 +1,48 @@
+/**
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/**
+ Container for the training data.
+ */
+internal struct BaseTrainingDataObjects: Codable, Equatable {
```

Comment:
This is crazy -- all kinds of "Base" models in here.  We need to take a hard look at the API Doc to see if we can fix this.